qa-tests-backend/scripts/run-compatibility.sh

```diff
@@ -82,6 +82,11 @@ _compatibility_test() {
 
     export CLUSTER="${ORCHESTRATOR_FLAVOR^^}"
 
+    verify_scannerV2_deployed "stackrox"
+    verify_scannerV4_deployed "stackrox"
```

Comment:
Do I understand correctly that we want to have both of them for all compatibility tests?
Do single tests pick which scanner is used actively? How do we know which one is actively used by Central?

Reply:
I just looked through our compatibility tests and it appears to be only `CertExpiryTest` which requires Scanner to be available, in order to read the scanner-tls secret, in order to compare it to the scanner cert expiry stored in central.

Which I think means that we would be okay only asserting against scanner v2 which takes less time as noted by Matthias.